bayesian/utility/v2/ca_response_builder.py

```diff
@@ -351,3 +374,87 @@ def get_cve_maps(self):
                 fixed_in=cve.get('fixed_in', [])
             ))
         return cve_list
+
+
+class CABatchResponseBuilder(ComponentResponseBase):
+    """Response builder for Component Analyses v2 Batch."""
+
+    def __init__(self, ecosystem):
+        """Batch CA Response Builder."""
+        super().__init__(ecosystem, None, None)
+
+    def generate_recommendation(self, package_graph_response: Dict) -> Dict:
+        """Generate recommendation for the package+version.
+
+        Main function to generate recommendation response.
+
+        :return: Json Response
+        """
+        logger.info("Generating Recommendation")
+        self.version = self.get_version(package_graph_response)
+        self.package = self.get_package(package_graph_response)
+        latest_non_cve_versions: List[str] = package_graph_response.get('package', {}).get(
+            'latest_non_cve_version', [])
+        self._cves = list(map(lambda x: x, package_graph_response.get('cve')))
```

Comment:
Do you need this lambda, it is not changing the value.

Reply:
Ack!